### PR TITLE
Vreijs/astrology

### PR DESCRIPTION
**Pull request by vreijs**
I think it is ready for a PR. I tried to contact teh feature requester but no avail yet. If he comes back, lets see if more is needed (but at this moment I think he has all the functionality needed). But I am not in astrology... Lets see how far we get with this 2.08 relaease, so we can also give feedback to SE people about 'open' issues. Found a minor issues around Section10's text...

### CODE REVIEW

**Comment by rstub** [comment]
Cool, lots of new functions :-) I haven’t looked at it closely yet, though. Anything else you want to add at the moment?

**Comment by vreijs** [comment]
I thought lets do also the House functions. The problem is that I don't understand anything of them, so I don't have a feel that the results are ok. Just assume that I implemented R correctly and that it matches the SE output. I hope that the feature requester will come back at some time (but he already said yesterday that he has no time at this momet). So lets progress in publicising it.

**Comment by vreijs** [comment]
I have now most functions implemented. A few eclipse functions (section 6) and functions from section 5 and 7 are still to be done, but I will not do that in this release.

File: src/swephR.cpp [hunk 1]
```diff
@@ -833,6 +1058,7 @@ void set_topo(double longitude, double lat, double height) {
 //'   \item{swe_day_of_week()}{Determine day of week from Julian day number.}
 //' }
 //' @param jd  Julian day number as numeric vector (day)
+//' @examples
```
Comment (rstub) [inline src/swephR.cpp]: This line looks spurious. I think it can be simply removed.
```suggestion
```

**Comment by rstub** [comment]
> I have now most functions implemented. A few eclipse functions (section 6) and functions from section 5 and 7 are still to be done, but I will not do that in this release.

Great. Any suggestions for version number? Should we go for 0.2.0, since we have new upstream and many more supported functions?

**Comment by vreijs** [comment]
Lets indeed go with 0.2.0 and I removed also the @example line.

I see the number of patches is quite reduced. Only two left. But is this one 15_uint_crc32.diff to be mentioned to the SE authors? Let me know, as I want to provide feedback to them.
The ERR -> SE_ERR  would be handy to get implemented. ERR is a little to generic;-).

**Comment by vreijs** [comment]
Thanks.

**Pull request merged**